tests/processing/test_processor.py

```diff
@@ -545,3 +548,40 @@ def test_dlq() -> None:
     processor._run_once()
 
     assert dlq_policy.producer.produce.call_count == 1
+
+
+def test_healthcheck(tmpdir: py.path.local) -> None:
```

Comment:
Can we move this test out of the processor too?

Reply:
I want to test processor + healthcheck e2e. testing in isolation makes no sense. where do you want me to put the test?

Reply:
Ah I see. The rest of the strategies are only tested in isolation so I was thinking under tests/processing/strategies with the rest. This is fine though.